documentation/writing_plans.md

```diff
@@ -822,11 +822,21 @@ to the log file.
 
 ### Puppet log functions
```

Comment:
So is this section specifically for logging Puppety stuff in Bolt? Or for logging from plans in general?

Reply:
I was under the impression that it was for using specifically built-in puppet functions as a way to log from plans, does that sound right? I think there are other ways to log like `out::message` (which I guess technically doesn't log, but does output things) as well that aren't affected by the difference between Puppet vs. Bolt log levels

Reply:
Reading through the doc, it looks like this is just to log from plans in general. The section should really just tell people how to use plan logs and then link off to log.md. Let's move the comparison between puppet log files and Bolt log files to logs.md

Reply:
I think I took too long to refresh and missed your comment @barriserloth. I'll drop you a line tomorrow

Reply:
Sounds good! Thanks for the feedback so far, I appreciate the help